servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java

```diff
@@ -619,29 +623,18 @@ private static StreamingHttpService buildService(Stream<StreamingHttpServiceFilt
     }
 
     private static HttpExecutionStrategyInfluencer buildInfluencer(List<StreamingHttpServiceFilterFactory> filters,
```

Comment:
add `@SuppressWarnings("unchecked")` for the cast to `ExecutionStrategyInfluencer<HttpExecutionStrategy>`.

also is it the plan to have `StreamingHttpServiceFilterFactory extend ExecutionStrategyInfluencer<HttpExecutionStrategy>`? making what is an influencer more explicit is one of the original goals IIRC.